### PR TITLE
ci: choose correct threshold sensitive for e2e

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [x] Build or CI related changes
- [ ] Documentation content changes

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: .github/workflows/demo-integrations.yml [hunk 1]
```diff
@@ -70,26 +69,21 @@ jobs:
...
+        run: npm run concurrently "npm run cy2:run:localhost" "npm run cy2:run:localhost"
 
       - name: Run screenshot tests on demo/next
         continue-on-error: true
-        run: |
-          npm run concurrently -- \
-                         "npm run cy2:run:demo:next" \
-                         "npm run cy2:run:demo:next"
+        env:
+          CYPRESS_failOnSnapshotDiff: true # collect all diff files for lumberjack bot
```
Comment (vladimirpotekhin) [inline .github/workflows/demo-integrations.yml:78]: there is no mistake? seems like it should be false

Comment (splincode) [reply .github/workflows/demo-integrations.yml]: fixed

**Comment by splincode** [comment]
@waterplea @vladimirpotekhin please, review me)

**Review by vladimirpotekhin — APPROVED** [review]

**Comment by splincode** [comment]
@waterplea please, review me

**Pull request closed without merging**